lib/dal.js

```diff
@@ -225,6 +225,16 @@ module.exports = utils.Class.extend({
     return this.query(query, callback);
   }
 
+, lock: function lock(mode, callback) {
+    if (typeof mode == 'function'){
+      callback = mode;
+      mode = null;
+    }
+
+    var query = 'LOCK TABLE ' + this.table + (mode ? ' IN ' + mode + ' MODE' : '');
```

Comment:
This is good for now. At some point, I'll want to add a `lock` query type that supports the full lock command syntax